builtin/providers/profitbricks/data_source_datacenter_test.go

```diff
@@ -0,0 +1,31 @@
+package profitbricks
+
+import (
+	"github.com/hashicorp/terraform/helper/resource"
+	"regexp"
+	"testing"
+)
+
+func TestAccDataSourceDatacenter_basic(t *testing.T) {
```

Comment:
This one is tricky in order to get a match we need to have a data center. I'm not sure how to accomplish that/